rules/aws_cloudtrail_rules/aws_ec2_startup_script_change.yml

```diff
@@ -0,0 +1,71 @@
+AnalysisType: rule
+Description: 'Detects changes to the EC2 instance startup script. The shell script will be executed as root/SYSTEM every time the specific instances are booted up. '
+DisplayName: AWS EC2 Startup Script Change
+Enabled: false
+Filename: aws_ec2_startup_script_change.py
+Reports:
+    MITRE ATT&CK:
+        - T1059
```

Comment:
The entries in `Reports.MITRE ATT&CK` should be `TAxxxx:Txxx` ( ta == tactic,  t== technique ). [example from another aws detection](https://github.com/panther-labs/panther-analysis/blob/master/rules/aws_cloudtrail_rules/aws_kms_cmk_loss.yml#L16)